### PR TITLE
Develop - add progress bar via tqdm

### PR DESCRIPTION
**Pull request by kidpixo**
I already merged latest commits from master in my develop.

### CODE REVIEW

File: comic_dl/globalFunctions.py [hunk 1]
```diff
@@ -78,10 +77,11 @@ def downloader(self, image_ddl, file_name, referer, directory_path, **kwargs):
                         # raw.senmanga doesn't return content-length. So, let's just assume 1024.
                         if total_length is None:
                             total_length = 1024
-
-                        for chunk in progress.bar(r.iter_content(chunk_size=1024),
-                                                  expected_size=(int(total_length) / 1024) + 1,
-                                                  hide=False):
+                        import tqdm
```
Comment (Xonshiz) [inline comic_dl/globalFunctions.py]: Please push this `import` to the top. Don't use any imports in between.

File: comic_dl/manga_eden/mangaChapterDownload.py [hunk 2]
```diff
@@ -42,7 +42,9 @@ def __init__(self, page_id, download_directory, **kwargs):
 
         globalFunctions.GlobalFunctions().info_printer(self.manga_name, self.chapter_number)
 
-        for image in self.image_links:
+        # for image in self.image_links:
+        import tqdm
```
Comment (Xonshiz) [inline comic_dl/manga_eden/mangaChapterDownload.py]: Push this `import` to the top as well.

**Comment by kidpixo** [comment]
Sorry,
My bad ugly development code. We can clean up before pushing to master
again. Next 2 weeks I'm on the beach with family and no computing power
(only mobile phone), leave it open here and I'll take care of it. Happy
holidays!

**Comment by Xonshiz** [comment]
I'd rather leave it open and fix things right away. I tend to forget things. Enjoy your holidays and we'll merge this later on. Happy Holidays!

**Comment by kidpixo** [comment]
Good point, I tend to forget too.
I must just remember to pull next time before tinkering!
Best.

On Jul 31, 2018 14:27, "Dhruv Kanojia (Xonshiz)" <notifications@github.com>
wrote:

I'd rather leave it open and fix things right away. I tend to forget
things. Enjoy your holidays and we'll merge this later on. Happy Holidays!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Xonshiz/comic-dl/pull/159#issuecomment-409202807>, or mute
the thread

**Comment by Xonshiz** [comment]
There seem to be some merge conflicts in this pull request. Kindly check on your end and push your changes again.

**Comment by darodi** [comment]
@Xonshiz , @kidpixo 
those changes have already been merged in #162 , this is a duplicate

except :
- Dockerfile
- ReadMe.md
- setup.py

...but those weren't really linked to tqdm

**Comment by Xonshiz** [comment]
Thanks for the update @darodi . @kidpixo , could you please make a new Pull Again with these 3 changes that @darodi  mentioned?

**Comment by dsanchezseco** [comment]
@Xonshiz I think this can be closed as is done in one of the previous PR from @darodi

**Pull request closed without merging**